### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Voiceover: MAS4.2.1: The Expand/collapse state of the "collapse panel" button is not being announced by the VoiceOver

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#435]
### Steps to Reproduce

1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations/Time Profiler" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Allocations/Time Profiler" Profiling page.
4. Navigate to the Collapse Panel(i.e., RHS in the profiler window) as shown in the attachment.
5. Observe the VoiceOver behavior for this button.
### Expected Behavior
The change in the state(i.e., Expand/collapse) should be announced by the VoiceOver.
### Actual Behavior
The Expand/collapse state of the collapse panel button is not being announced by the VoiceOver.
Note: Once User expands/collapse the Collapse panel, the  VoiceOver announces "Press Collapse Panel" instead of "Collapse panel expanded/collapsed".
### User Impact
The User won't be able to know the actual state of the panel until its expand/collapse state is not being announced
### Screenshot
![voiceover_profilermac2](https://user-images.githubusercontent.com/22956130/41714602-7ac5b33e-756e-11e8-9be2-2a3359967342.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following tags to this bug:
A11yMAS
Accessibility
MAS4.2.1
Xamarin-Profiler-MAC
Wipro
